README.md

```diff
@@ -102,182 +106,44 @@ cargo test
 
 ## Contributing
 
-I am not currently able to handle contributions. After the 0.1 release in April I will be looking for help in a few areas:
+We are actively seeking contributors! Please join the [Discord](https://discord.gg/qJCsKJeyZv) and come help out!
 
-**Design:** The tool must look and feel good and I am not a designer. I would love contributions in the form of:
+Most especially, we need help in the following areas:
+
+**Design:** The tool must look and feel good and we are not designers. We would love contributions in the form of:
 - Advice, mockups, or tailwindcss examples of how to make different elements look and behave better.
```

Comment:
could drop the full-stop (period) at the end of this bullet point as the other sentences don't have periods.

Reply:
line 114